content/momentum/navigation.yml

```diff
@@ -513,8 +513,6 @@
           title: Using Complex Templates
     - link: /momentum/4/mta-sts
       title: MTA-STS support
-    - link: /momentum/4/modules/dane
-      title: DANE support
     - link: /momentum/4/http-api-stats
```

Comment:
I prefer to keep it in navigation.

Reply:
As a major feature, I would prefer it's easy to find when the customer browse through the support doc, instead of depending on search. In "navigation", the user will see it listed under "Introduction to Momentum", same as MTA-STS. We did not do it for existing features, but I feel it's a good idea to have an entry point for new major features in the main index list.